EventFilter/Utilities/src/FedRawDataInputSource.cc

```diff
@@ -48,6 +48,21 @@
 
 using namespace evf::FastMonState;
 
+class IdleSourceSentry {
```

Comment:
This class is repeated from `DAQSource.cc`. Given that the class definitions are the same, this is technically ok, but if only one of them would be changed in the future, that would lead to ODR violation. How about moving the code of `IdleSourceSentry` into a header `#include`d from both source files? Or moving both class definitions into an anonymous namespace?

Reply:
Yes, I had doubts about that. I aded a new header (which, out of scope of this PR, could host also other shared classes between two sources).